### PR TITLE
Document describe.one usage

### PR DESCRIPTION
**Pull request by james-stocks**
As suggested by @aaronlippold in community Slack

Fixed section indenting in the same commit.

Signed-off-by: James Stocks <jstocks@chef.io>

### CODE REVIEW

File: docs/dsl_inspec.md [hunk 1]
```diff
@@ -73,6 +75,12 @@ describe.one do
 end
 ```
 
+#### `describe.one` usage notes
+
+* InSpec considers a `describe.one` block as passing if a `describe` nested within has all assertions passing. That is, it needs an entire `describe` block to pass and not just single assertions.
```
Comment (clintoncwolfe) [inline docs/dsl_inspec.md]: "not just a single assertion"

**Review by clintoncwolfe — CHANGES_REQUESTED** [review]
Thanks, this is much better! I caught one minor grammo.

File: docs/dsl_inspec.md [hunk 2]
```diff
@@ -73,6 +75,12 @@ describe.one do
 end
 ```
 
+#### `describe.one` usage notes
+
+* InSpec considers a `describe.one` block as passing if a `describe` nested within has all assertions passing. That is, it needs an entire `describe` block to pass and not just a single assertion.
```
Comment (stevendanna) [inline docs/dsl_inspec.md]: It may be the result of too little coffee, but I found this and some of the following sentences hard to follow on first reading.  I think some of it might be the result of the tense of this sentence:

```suggestion
* A `describe.one` block passes if one of its nested `describe` blocks have all assertions passing. That is, it needs an entire `describe` block to pass and not just a single assertion.
```

**Review by stevendanna — APPROVED** [review]
Nice improvements.  I left a minor comment on the one bullet point that I found a bit difficult to parse, but feel free to take or leave the suggestion as I'm not sure I've actually made it better.

File: docs/dsl_inspec.md (hunk 2 above)
Comment (james-stocks) [reply docs/dsl_inspec.md]: Thanks Steven, that is a neater way to write it. I think `has all assertions` is correct because a singular block is the subject.

Comment (stevendanna) [reply docs/dsl_inspec.md]: @james-stocks Yes, sorry, I was playing around with a few different ideas and let that error creep in. `has` is definitely correct here.

**Review by clintoncwolfe — APPROVED** [review]
Thanks!

**Review by mjingle — APPROVED** [review]
